plugins/modules/onepassword_info.py

```diff
@@ -193,9 +191,7 @@ def _run(self, args, expected_rc=0, command_input=None, ignore_errors=False):
             args += [to_bytes("--session=") + self.token]
 
         command = [self.cli_path] + args
-        p = Popen(command, stdout=PIPE, stderr=PIPE, stdin=PIPE)
-        out, err = p.communicate(input=command_input)
-        rc = p.wait()
+        rc, out, err = module.run_command(command, data=command_input, check_rc=False)
```

Comment:
You must pass `binary_data=True` since `command_input` is either `None` or `bytes`. Otherwise the module will crash when `AnsibleModule.run_command` tries to append `"\n"` to a `bytes` string. Also the code expects `out` and `err` to be `bytes`, so `encoding=None` should be passed.
```suggestion
        rc, out, err = module.run_command(command, data=command_input, check_rc=False, binary_data=True, encoding=None)
```